From f538e79798876d396cd463fc1497d1ace590439b Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 27 Apr 2023 13:18:14 +0800
Subject: [PATCH 54/68] Revert "drm/msm/hdmi: Add missing check for
 alloc_ordered_workqueue"

This reverts commit 1bab31a0969ca4ac90907a5d3b44af104229eafd.
---
 drivers/gpu/drm/msm/hdmi/hdmi.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/msm/hdmi/hdmi.c b/drivers/gpu/drm/msm/hdmi/hdmi.c
index bee208773beec..efb14043a6ec4 100644
--- a/drivers/gpu/drm/msm/hdmi/hdmi.c
+++ b/drivers/gpu/drm/msm/hdmi/hdmi.c
@@ -264,10 +264,6 @@ static struct hdmi *msm_hdmi_init(struct platform_device *pdev)
 	pm_runtime_enable(&pdev->dev);
 
 	hdmi->workq = alloc_ordered_workqueue("msm_hdmi", 0);
-	if (!hdmi->workq) {
-		ret = -ENOMEM;
-		goto fail;
-	}
 
 	hdmi->i2c = msm_hdmi_i2c_init(hdmi);
 	if (IS_ERR(hdmi->i2c)) {
-- 
2.25.1

